.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 config.json
 failure.png
+.vscode/
+.DS_Store
```

Comment:
This files came from developer's computer, I prefer add it in global gitignore config and keep clean the project gitignore with only project related files

Reply:
Sure no worries 👍